Dokan/Dokan/Extension/UIImage+Extension.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIImage+Extension.swift
+//  Dokan
+//
+//  Created by raniazeid on 21/07/2022.
+//
+
+import Foundation
+import Kingfisher
+import UIKit
+
+
+extension UIImageView{
+    
+    func setImage(with urlString: String) {
+        let url = URL(string: urlString)
+        self.kf.setImage(with: url,
```

Comment:
I would highly recommend exposing the KF `setImage` parameters and set default values for them in `setImage(with:)` signature. It would help for more customizations if needed, for example changing `placeholder` image.

Reply:
thanks for your review i will take it into consideration next time